src/slice.js

```diff
@@ -8,19 +8,27 @@ import {validateSliceParams} from './lib/validate-input.js';
  * @param {Tensor} input
  * @param {Array} starts
  * @param {Array} sizes
+ * @param {MLSliceOptions} options
  * @return {Tensor}
  */
-export function slice(input, starts, sizes) {
+export function slice(input, starts, sizes, {strides} = {}) {
   validateSliceParams(...arguments);
-  const outputShape = sizes;
+  strides = strides ?? new Array(input.rank).fill(1);
+  const outputShape = input.shape.slice();
+
+  for (let dimensionIndex = 0; dimensionIndex < input.rank; ++dimensionIndex) {
+    outputShape[dimensionIndex] = Math.floor(sizes[dimensionIndex] / strides[dimensionIndex]) +
```

Comment:
Here you can use `Math.ceil(sizes[dimensionIndex] / strides[dimensionIndex])`, it works the same as `Math.floor(sizes[dimensionIndex] / strides[dimensionIndex]) + Number(sizes[dimensionIndex] % strides[dimensionIndex] != 0)`.